### PR TITLE
Add  PyMsql Plugin

### PR DESCRIPTION
**Pull request by alonelaval**

### CODE REVIEW

File: skywalking/plugins/sw_pymysql/__init__.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+def install():
+    # noinspection PyBroadException
+    try:
+        from pymysql.cursors import Cursor
+
+        _execute = Cursor.execute
+
+        def _sw_execute(this: Cursor, query, args=None):
```
Comment (kezhenxu94) [inline skywalking/plugins/sw_pymysql/__init__.py:35]: we can provide another option to allow the users to collect the `args`, though that can be done in another PR as an enhancement, if you want to do that, refer to `plugin.mysql.trace_sql_parameters` and `plugin.mysql.sql_parameters_max_length` in https://github.com/apache/skywalking/blob/cc9ea271a720e8def161a583595f75da9bda77a5/docs/en/setup/service-agent/java-agent/README.md#table-of-agent-configuration-properties

File: tests/plugin/sw_pymysql/services/provider.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+from skywalking import agent, config
+
+if __name__ == '__main__':
+    config.service_name = 'provider'
+    config.logging_level = 'DEBUG'
+    agent.start()
+
+    from flask import Flask, jsonify
+    import pymysql.cursors
```
[the remark this answers is not in the document]
Comment (alonelaval) [reply tests/plugin/sw_pymysql/services/provider.py:28]: ok

File: skywalking/plugins/sw_pymysql/__init__.py (hunk 1 above)
Comment (alonelaval) [reply skywalking/plugins/sw_pymysql/__init__.py:35]: > 

> we can provide another option to allow the users to collect the `args`, though that can be done in another PR as an enhancement, if you want to do that, refer to `plugin.mysql.trace_sql_parameters` and `plugin.mysql.sql_parameters_max_length` in https://github.com/apache/skywalking/blob/cc9ea271a720e8def161a583595f75da9bda77a5/docs/en/setup/service-agent/java-agent/README.md#table-of-agent-configuration-properties

ok

**Comment by alonelaval** [comment]
> Looks good to me in codes and locally test, thanks @alonelaval very much, please remember to update the component in the main repo, we may need mapping this time

good vpn...LOL

**Pull request merged**